### ISSUE TITLE
rc-telephony-session-notify event does not trigger in 1.9.3+

### ISSUE THREAD

**Issue by sagearslan** [ringcentral/ringcentral-embeddable#738]
Updated to 1.9.3, the `rc-telephony-session-notify` event does not trigger.

Previous usage in 1.8 like so:

```
window.addEventListener('message', (e) => {
  const data = e.data;
  if (data) {
    switch(data.type) {
      case 'rc-telephony-session-notify':
        // do something with `data` here
        break;
      // ...
    };
  }
});
```

**Comment by embbnux** [comment]
fixed #737

**Comment by embbnux** [comment]
closing this as fixed at #737
